src/index.js

```diff
@@ -219,7 +220,7 @@ function renderToString(vnode, context, opts, inner, isSvgMode, selectValue) {
 	s = `<${nodeName}${s}>`;
 	if (String(nodeName).match(/[\s\n\\/='"\0<>]/)) throw new Error(`${nodeName} is not a valid HTML tag name in ${s}`);
 
-	let isVoid = String(nodeName).match(VOID_ELEMENTS);
+	let isVoid = String(nodeName).match(opts.voidElements || DEFAULT_VOID_ELEMENTS);
```

Comment:
This PR is awesome :raised_hands: 

We can combine the two by always checking for `DEFAULT_VOID_ELEMENTS` and checking for `opts.voidElements` if it was defined. Otherwise defining a custom element requires the user to add all default void elements in their regex.

Reply:
I first had 2 config options (`voidElements` and `additionalVoidElements`) where the later did exactly what you mentioned. With the former one has the full flexibility with the drawback you mentioned. Not sure what is best here.

Reply:
We could export the default_void_elements, this way the user has access to them and can expand on them.

Reply:
Or something like:

```js
let isVoid = nodeName.match(opts.voidElements) || nodeName.match(DEFAULT_VOID_ELEMENTS)
```

Reply:
@marvinhagemeister the only drawback I saw there is that one is not able to completely opt out of void elements. But that might be a too crazy usecase. 

I‘ll adjust the PR soonish

Reply:
Seeing that Preact's main focus is the web, I think we're good with that :+1: